README.md

```diff
@@ -1 +1,47 @@
-# prow-draft-pullrequest-plugin
\ No newline at end of file
+# Prow Draft Pull Request Plugin
+
+## Introduction
+The goal of the plugin is to restrict the number of prow jobs that normally runs on pull request. Since /ok-to-test cannot be applied on trusted members pull request in prow CI/CD.
```

Comment:
```suggestion
This plugin restrict the initial number of prow jobs that normally runs on pull request, since `/ok-to-test` can't be applied on trusted member's pull request in prow CI.
```